qchem/pennylane_qchem/qchem/obs.py

```diff
@@ -206,55 +206,61 @@ def spin2(electrons, orbitals, mapping="jordan_wigner", wires=None):
 
     table = _spin2_matrix_elements(sz)
 
-    return observable(table, init_term=3 / 4 * electrons, mapping=mapping, wires=wires)
+    return observable([table], init_term=3 / 4 * electrons, mapping=mapping, wires=wires)
 
 
-def observable(me_table, init_term=0, mapping="jordan_wigner", wires=None):
+def observable(me_tables, init_term=0, mapping="jordan_wigner", wires=None):
```

Comment:
Are there any suggestions for an alternative to `me_tables`? The "me" in `me_tables` really sticks out to me!
Could it be something like `elems` and we explain more in the docstring?

Reply:
even something like `matrix_elements` seems more descriptive to me. I've always found the 'tables' suffix somewhat redundant